zaxgrpc/zaxgrpc_test.go

```
@@ -0,0 +1,187 @@
+type fakeStream struct {
+	grpc.ServerStream
+	ctx context.Context
+}
+
+func (f *fakeStream) Context() context.Context {
+	return f.ctx
+}
+
+func TestStreamExposesEnrichedContext(t *testing.T) {
+	logger, recorded := newTestLogger()
+	info := &grpc.StreamServerInfo{FullMethod: "/svc/Stream"}
```

golangci-lint (2.12.2)

[error] string `/svc/Stream` has 3 occurrences, make it a constant

(goconst)